scripts/cci-workflow-results.ts

```diff
@@ -0,0 +1,95 @@
+import { join } from "path";
+import { getOldJobNameWithoutSuffixes, getTestFiles, getTimingsFromJobsData, REPO_ROOT, saveTestTimings, saveWorkflowResults, saveWorkflowResultsHTML } from "./cci-utils";
+import axios from 'axios';
+
+const getWorkflowData = async () => {
+    const result = await axios.get(`https://circleci.com/api/v2/workflow/${process.env.CIRCLE_WORKFLOW_ID}/job`, {
+        headers: {
+            "Circle-Token": process.env.CIRCLECI_TOKEN 
+        }
+    });
+    return result.data;
+}
+const getJobData = async (jobId: string) => {
+    const result = await axios.get(`https://circleci.com/api/v2/project/github/aws-amplify/amplify-cli/job/${jobId}`, {
+        headers: {
+            "Circle-Token": process.env.CIRCLECI_TOKEN 
+        }
+    });
+    return result.data;
+}
+const getJobArtifacts = async (jobId: string) => {
+    const result = await axios.get(`https://circleci.com/api/v2/project/github/aws-amplify/amplify-cli/${jobId}/artifacts`, {
+        headers: {
+            "Circle-Token": process.env.CIRCLECI_TOKEN 
+        }
+    });
+    return result.data;
+}
+const getJobTests = async (jobId: string) => {
+    const result = await axios.get(`https://circleci.com/api/v2/project/github/aws-amplify/amplify-cli/${jobId}/tests`, {
+        headers: {
+            "Circle-Token": process.env.CIRCLECI_TOKEN 
+        }
+    });
+    return result.data;
+}
+
+
+const runit = async () => {
```

Comment:
todo: add js docs

This function runs after a workflow has completed, and it aggregates the results of all the failed tests into a single HTML page/JSON doc that you can locate in the 'wait_for_all' job.
Sample: https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/14097/workflows/792d5fbd-f763-435a-b00a-c325c225699b/jobs/663225/artifacts